### DISCUSSION TITLE
Show English translations if no local translations

### DISCUSSION THREAD

**Discussion by bitdom8** [ivanhofer/typesafe-i18n#340] [Q&A] [answered]
Created a keys and values of some tables, however, some of them have no translations and it omits and leaves blank. Can we show English if no translations?

```
	{#if product.datajson}
		<div class="tableFixHead">
			<table>
				<thead>
					<tr><th>{$LL.ptitle.Indicator()}</th><th>{$LL.ptitle.Value()}</th></tr>
				</thead>
				<tbody>
					{#if Object.keys(product.datajson).length > 0}
						{#each Object.keys(product.datajson) as key}
							<tr
								><td> {$LL.ptitle[String(key).substring(1, key.length - 1)]()} 
								</td><td
									>{String(product.datajson[key])}</td
								></tr
							>
						{/each}
					{/if}
				</tbody>
			</table>
		</div>
	{/if}
```

**Comment by ivanhofer** [comment, marked as answer]
Hi @bitdom8,
if you don't translate every entry for some locale, you could fallback to the english string by writing it like in this example:

https://github.com/ivanhofer/typesafe-i18n#i-have-two-similar-locales-only-a-few-translations-are-different-but-i-dont-want-to-duplicate-my-translations

**Reply by bitdom8** [reply]
Thanks @ivanhofer , that will nicely do it
